packages/models-library/src/models_library/api_schemas_webserver/users.py

```diff
@@ -141,7 +141,7 @@ def from_domain_model(
 class MyProfilePatch(InputSchemaWithoutCamelCase):
     first_name: FirstNameStr | None = None
     last_name: LastNameStr | None = None
-    user_name: Annotated[IDStr | None, Field(alias="userName")] = None
+    user_name: Annotated[IDStr | None, Field(alias="userName", min_length=4)] = None
```

Comment:
Question, since I see no tests relative to it.
- what happens when I register and my name or surname are 2/3 characters long, how is the user_name autogenerated?

Reply:
1. There *are* tests. Please check the changes `packages/models-library/tests/test_api_schemas_webserver_users.py`
2.  The auto-generation do guarantee this constraint. Please check `packages/postgres-database/src/simcore_postgres_database/utils_users.py`

Reply:
Thnaks. All good then!